boilerplate.js

```diff
@@ -92,7 +92,7 @@ SOFTWARE.
   DREEM_SERVER_AVAILABLE = false;
   var compatibleServerVersions = ["1.0.0", "1.0.1"];
   var request = new XMLHttpRequest();
-  request.open('GET', DREEM_ROOT + 'version', false);
+  request.open('GET', DREEM_ROOT + 'info', false);
```

Comment:
The server API has changed a little, /version is now deprecated.  This change isn't important for the rest of the PR.